CHANGELOG.md

```diff
@@ -8,12 +8,15 @@ CHANGELOG
 
 - Enable support for NICE DCV in GovCloud regions.
 - Enable support for AWS Batch scheduler in GovCloud regions.
+- Add ``-r/-region`` arg to ``pcluster configure``. If this arg is provided, configuration will skip region selection.
 
 **CHANGES**
 
 - Upgrade image used by CodeBuild environment when building container images for Batch clusters.
 - Enable queue resizing on update without requiring to stop the compute fleet. Stopping the compute fleet is only
   necessary when existing instances risk to be terminated.
+- Change default region from us-east-1 to whatever region written in aws config file.
+- Require ``aws_region_name`` under ``[aws]`` section
```

Comment:
I'd remove this requirement, because we can use environment variables or region from AWS configuration.

Reply:
Done